src/haz3lschool/SyntaxTest.re

```diff
@@ -130,6 +143,12 @@ let rec var_applied = (name: string, uexp: Term.UExp.t): bool => {
   };
 };
 
+/*
+ Goes into the pattern of a let expression and finds any mention of a variable
```

Comment:
This is confusing to me, maybe say which function it is used as a helper in and comment it relative to that?